client/src/js/seedData.js

```diff
@@ -4969,7 +4969,7 @@ var seeds = {
         "author": "Ann L.H; Marvin Rosemberg",
         "year": 1980,
         "image_name": "annlhmarvinrosemberg_sintitulo.png",
-        "": ""
```

Comment:
What is this row? Looking at the existing seed data, it looks like it is inserting an empty property for each record. Probably because an empty column is in the source data?